recipes/asv/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "asv" %}
+{% set version = "0.2" %}
+{% set sha256 = "4d51a63c1604e513ae38d3d0fd420815d27e50239bfb6e8e43d5e0907745d649" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sphinx
+    - sphinx_bootstrap_theme
```

Comment:
Elevated to the feedstock.

xref: https://github.com/conda-forge/sphinx-bootstrap-theme-feedstock/issues/2